web/src/types/index.ts

```diff
@@ -8,7 +8,6 @@ export interface Dandiset {
   identifier: string,
   created: string,
   modified: string,
-  // TODO these versions are girderized, they should have type Version
```

Comment:
It isn't possible to set these to type `Version` since `Version` is defined after `Dandiset` and has a field of type `Dandiset`.

Reply:
Something like https://stackoverflow.com/a/48419531 might help. This feels like there is a solution, especially because both definitions are in the same file.

Reply:
How about this? https://github.com/dandi/dandiarchive/pull/782/commits/d7c3167340f223f8b973e614f25925022ae63c12